src/fmu/sumo/sim2sumo/common.py

```diff
@@ -193,6 +194,8 @@ def prepare_list_for_sendoff(datatype, simconfig, datafiles, paths, grid3d):
     logger.debug("Submodules to extract with: %s", submods)
     outdict = {}
     options = simconfig.get("options", {"arrow": True})
+    if isinstance(submods, list):
+        submods = {sub: options for sub in submods}
```

Comment:
Will this make the `try/except` on line 205-208 unnecessary?
Seems like `submods` will be forced to always be a dict with `.values() = options`?

Reply:
Think one of the tests are failing because of this change.
`filter_options` one line 210 expects `suboptions` to be a dict, but with this change the `.values()` on line 206 works and so `suboptions` turns to a list.